awsconfigs/apps/ack-sagemaker/kustomization.yaml

```diff
@@ -0,0 +1,67 @@
+# cluster scoped ack installation 
```

Comment:
maybe

- [ ] - change dir name to sagemaker-controller
- [ ] - move under common
- [ ] - put everything from upstream under base with a kustomization.yaml
- [ ] - try using kustomize patch for deployment to directly consume from configmap https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/common/aws-alb-ingress-controller/base/deployment_patch.yaml
- [ ] - remove ack.yaml, .DS_store in multiple places
- [ ] - RDS and S3 only deployment bases to consume from new pipeline base

Reply:
Thanks 👍🏻